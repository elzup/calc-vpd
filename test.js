const m = require('.')

test('snapshot', () => {
  expect(m({ tmp: 29.2, hmd: 76.5 })).toMatchSnapshot()
  expect(m({ tmp: 29, hmd: 75 })).toMatchSnapshot()
  expect(m({ tmp: 10, hmd: 92 })).toMatchSnapshot()
})

test('invalid argument', () => {
  expect(() => m({ hmd: 92 })).toThrowErrorMatchingSnapshot()
  expect(() => m({ tmp: 10, hmd: 'abc' })).toThrowErrorMatchingSnapshot()
})
