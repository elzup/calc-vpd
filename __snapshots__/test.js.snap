// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`invalid argument 1`] = `"Expected a { tmp: number, hmd: number }, got undefined number"`;

exports[`invalid argument 2`] = `"Expected a { tmp: number, hmd: number }, got number string"`;

exports[`snapshot 1`] = `
Object {
  "swv": 29.080252344,
  "vp": 40.518038231,
  "vpd": 6.833859301,
}
`;

exports[`snapshot 2`] = `
Object {
  "swv": 28.765370453,
  "vp": 40.052795772,
  "vpd": 7.191342613,
}
`;

exports[`snapshot 3`] = `
Object {
  "swv": 9.410297414,
  "vp": 12.278920335,
  "vpd": 0.752823793,
}
`;
